openmc/plotter.py

```diff
@@ -47,14 +58,20 @@
 PLOT_TYPES_LINEAR = {'nu-fission / fission', 'nu-scatter / scatter',
                      'nu-fission / absorption', 'fission / absorption'}
 
+# Minimum and maximum energies for plotting (units of eV)
+_MIN_E = 1.E-5
+_MAX_E = 20.E6
```

Comment:
Also, lower-case e is used in scientific notation elsewhere in the Python API and scripts, so please stay consistent (sorry for being nitpicky!)